rastervision_pytorch_learner/rastervision/pytorch_learner/learner.py

```diff
@@ -664,7 +664,11 @@ def normalize_input(self, x: Tensor) -> Tensor:
             the same tensor that has been scaled to [0-1].
 
         """
-        return x.float() / 255.0
+        if x.dtype == np.uint8 or x.dtype == np.uint16 or x.dtype == np.uint32:
```

Comment:
x is a torch tensor so you need to use torch.uint8, etc or the equality check won't work.

Reply:
Hmmm ... it seems as though `torch.uint16` and `torch.uint32` do not exist.